libbeat/docs/release.asciidoc

```diff
@@ -34,6 +36,8 @@ upgrade.
 * <<release-notes-7.0.0-beta1>>
 * <<release-notes-7.0.0-alpha2>>
 * <<release-notes-7.0.0-alpha1>>
+* <<release-notes-6.8.11>>
+* <<release-notes-6.8.10>>
```

Comment:
Are these sections already in the Changelog.asciidoc? You might need to copy 6.8.10 and 6.8.11 changelog to the 7.8, 7.x, and master branch.

Reply:
I suspected something was missing

Reply:
Every branch has a different changelog, even the master branch does not have all release notes versions, I will rid of that part

Reply:
> Every branch has a different changelog

The changelog in each branch must be held consistent. The changelog and documentation is generated from the release branch. This means that 7.8 release branch must include all changelogs for past 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, ... 6.8, ... releases

The 7.8 branch will contain all changelogs up to the most recent 7.8.x patch release.
The 7.9 branch will contain all changelogs up to the most recent 7.9.x patch release.

The changelogs in the 7.x and master branch need to be kept in sync with the actual release branches.

If the changelogs in the branches differ, then we're missing the required updates in some branches.